pygments/lexers/_openedge_builtins.py

```diff
@@ -448,6 +457,7 @@
     'DATA-SOURCE-MODIFIED',
     'DATA-SOURCE-ROWID',
     'DATA-TYPE',
+    'DATE',
```

Comment:
this is not alphabetically sorted

Reply:
The whole list was not really 100% sorted correctly, i have fixed the sorting of the whole list now.